package/README.md

```diff
@@ -57,35 +57,42 @@ Here is an overview of the release process. See the section below for detailed i
+
+3. Tag the latest commit on the master branch. In this project, tags have form `X.Y.Z`, e.g. `2.4.0`, or `X.Y.Z-rcN` for release versions. Notice that there's no `v`at the beginning of the tag. Tags must be annotated, not lightweight tags. To create a tag, use git command (assuming that the master branch is checked out):
 
    ```
-   git tag -a -m "Release 2.4.0" 2.4.0
+   git tag -a -m "Release 2.5.0" 2.5.0
    ```
 
-3. Push the tag created in step 2 to esp8266/Arduino Github repository:
+4. Push the tag created in step 2 to esp8266/Arduino Github repository:
 
    ```
-   git push origin 2.4.0
+   git push origin 2.5.0
    ```
 
-4. Wait for Travis CI build for the tag to pass. Check that the new (draft) release has been created. Check that the boards manager package .zip file has been successfully uploaded as a release artifact.
+5. Wait for Travis CI build for the tag to pass. Check that the new (draft) release has been created. Check that the boards manager package .zip file has been successfully uploaded as a release artifact.
+
+6. Check that the package index downloaded from http://arduino.esp8266.com/stable/package_esp8266com_index.json contains an entry for the new version (it may not be the first one).
+
+7. Navigate to release list in Github, press "Edit" button to edit release description, paste release notes, and publish it.
 
-5. Check that the package index downloaded from http://arduino.esp8266.com/stable/package_esp8266_index.json contains an entry for the new version (it may not be the first one).
+8. In the issue tracker, remove "staged-for-release" label for all issues which have it, and close them. Close the milestone associated with the released version.
 
-6. Navigate to release list in Github, press "Edit" button to edit release description, paste release notes. Remove "draft" status of the release and publish it.
+9. Check that https://arduino-esp8266.readthedocs.io/en/latest/ has a new doc build for the new tag, and that "stable" points to that build. If a new build did not trigger, log into readthedoc's home and trigger it manually.
 
-7. In the issue tracker, remove "staged-for-release" label for all issues which have it, and close them. Close the milestone associated with the released version.
 
-8. Create a commit to the master branch, updating:
+10. Create a commit to the master branch, updating:
 
    * The version in platform.txt file. This should correspond to the version of the *next* milestone, plus `-dev` suffix. E.g. `2.5.0-dev`.
 
    * In main README.md:
 
-     - in "Contributing" section, update the "easy issues" link to point to the next milestone.
-
      - in "Latest release" section, change version number in the readthedocs link to the version which was just released.
 
+     - Replace any other instances of the version number in the doc.
+     
+     - Check that links to the latest readthedocs build are correct.
```

Comment:
Can be combined with the point above, e.g.

- in "Latest release" section, change version number in the readthedocs link to the version which was just released, and verify that the link works